roles/rsyslog/roles/output_roles/elasticsearch/defaults/main.yaml

```diff
@@ -77,7 +77,7 @@ rsyslog_conf_es_elasticsearch:
           }
 
           ruleset(name="try_es") {
-              if strlen($.omes!status) > 0 then {
+              if (strlen($.omes) > 0) and (strlen($.omes!status) > 0) then {
```

Comment:
Also these two cases:
```
              } else if strlen($.omes!_id) > 0 then {
```
and
```
              if strlen($.omes!_index) > 0 then {
```

Reply:
This is already in the code ...
https://github.com/linux-system-roles/logging/blob/059d70fe43ca78b497c87f606e1ed133cf373867/roles/rsyslog/roles/output_roles/elasticsearch/defaults/main.yaml#L97
https://github.com/linux-system-roles/logging/blob/059d70fe43ca78b497c87f606e1ed133cf373867/roles/rsyslog/roles/output_roles/elasticsearch/defaults/main.yaml#L104
@richm

Reply:
> This is already in the code ...

Not sure what you mean

You cannot check for `strlen($.omes!_id)` or `strlen($.omes!_index)` without first checking `strlen($.omes)`

Reply:
Sorry. I fixed it now @richm

Reply:
looks good